packages/server/lib/errors.js

```diff
@@ -900,6 +900,11 @@ const getMsgByType = function (type, arg1 = {}, arg2, arg3) {
       return stripIndent`\
         The \`experimentalShadowDomSupport\` configuration option was removed in Cypress version \`5.2.0\`. It is no longer necessary when utilizing the \`includeShadowDom\` option.
 
+        You can safely remove this option from your config.`
+    case 'EXPERIMENTAL_NETWORK_STUBBING_REMOVED':
+      return stripIndent`\
+        The \`experimentalNetworkStubbing\` configuration option was removed in Cypress version \`6.0.0\`. It is no longer necessary for using \`cy.http()\` (formerly \`cy.route2()\`).
```

Comment:
np: Maybe add a newline at the 2nd sentence. It reads a bit long in an expanded terminal.

```suggestion
        The \`experimentalNetworkStubbing\` configuration option was removed in Cypress version \`6.0.0\`. 
        It is no longer necessary for using \`cy.http()\` (formerly \`cy.route2()\`).
```

<img width="1356" alt="Screen Shot 2020-11-17 at 2 51 53 PM" src="https://user-images.githubusercontent.com/1271364/99364681-a9b81480-28e4-11eb-9c6a-f1c4eeec897e.png">